core/templates/span.h

```diff
@@ -149,16 +162,10 @@ constexpr int64_t Span<T>::rfind(const T &p_val, uint64_t p_from) const {
 }
 
 template <typename T>
-constexpr int64_t Span<T>::rfind_sequence(const Span<T> &p_span, uint64_t p_from) const {
+template <typename T1>
+constexpr int64_t Span<T>::rfind_sequence(const Span<T1> &p_span, uint64_t p_from) const {
 	for (int64_t i = p_from; i >= 0; i--) {
```

Comment:
should probably be `i = Minimum(p_from, size() - p_span.size())` or spans_equal might search out of bounds

Reply:
`Span` is designed to be a low-level API, much like C++ low level APIs. In that, it assumes the caller knows what they're doing, and passes valid arguments. 
If `Span` was defensive, and checked for the size against the parameter, it could measurably slow down `rfind_sequence` calls in a loop (which is something we actually do), because of the unnecessary check overhead.